WcaOnRails/app/models/person.rb

```diff
@@ -69,13 +69,9 @@ def to_jsonable(include_private_info: false)
       country_iso2: self.country_iso2,
     }
 
-    if include_private_info
-      json[:dob] = person.dob
-    end
-
     # If there's a user for this Person, merge in all their data,
     # the Person's data takes priority, though.
-    json = (user || User.new).to_jsonable.merge(json)
+    json = (user || User.new).serializable_hash.merge(json)
```

Comment:
The assignment is unnecessary, `(user || User.new).serializable_hash.merge(json)` should work, too.